buffer/src/main/java/io/netty/buffer/CompositeByteBuf.java

```diff
@@ -648,6 +660,20 @@ protected int _getUnsignedMedium(int index) {
     }
 
     @Override
+    protected int _getUnsignedMediumLE(int index) {
+        Component c = findComponent(index);
+        if (index + 3 <= c.endOffset) {
+            return c.buf.getUnsignedMediumLE(index - c.offset);
+        } else if (order() == ByteOrder.BIG_ENDIAN) {
+            return _getShortLE(index) & 0xFFFF |
```

Comment:
nit: consider putting this on 1 line and also consider using lowercase `f`s for consistency.